teuthology/config.py

```diff
@@ -7,6 +7,12 @@
     from collections import MutableMapping
 
 
+# Configuration constants
+_SYSTEM_CONFIG_PATH = '/etc/teuthology.yaml'
+_USER_CONFIG_PATH = '~/.teuthology.yaml'
+_ALT_PATH_ENV_VAR = 'TEUTHOLOGY_CONFIG'
+
```

Comment:
According to PEP8 there is no need to add _ (underscore) in the beginning of the constant name, may we have just `SYSTEM_CONFIG_PATH` and `USER_CONFIG_PATH`?
Also, I think to keep similar naming to env, like `CONFIG_PATH_ENV_NAME`.